### PR TITLE
Update yaml files to use seccomp GA syntax

### PR DESCRIPTION
**Pull request by pjbgf**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Seccomp became GA on v1.19 and its usage now uses fields instead of annotations. This PR migrates all yaml files inside this repository to the GA syntax.

**Which issue(s) this PR fixes**:

Relates to #91286

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/sig node
/area security

### CODE REVIEW

**Comment by liggitt** [comment]
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/93095/pull-kubernetes-node-e2e/1283657114006327296 flake is #93210

File: cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml [hunk 1]
```diff
@@ -64,9 +64,10 @@ spec:
       # This annotation ensures that fluentd does not get evicted if the node
       # supports critical pod annotation based priority scheme.
       # Note that this does not guarantee admission on the nodes (#40573).
```
Comment (liggitt) [inline cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml:66]: drop this comment, the related annotation no longer exists

**Comment by liggitt** [comment]
one comment, looks good otherwise. did you try creating these to make sure syntax was correct?

File: cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml (hunk 1 above)
Comment (spiffxp) [reply cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml:66]: This still needs to be addressed

**Comment by spiffxp** [comment]
/retest
Is this something we want to land for v1.19?

File: cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml (hunk 1 above)
Comment (pjbgf) [reply cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml]: Thanks for pointing it out, the comments are now removed.

**Comment by pjbgf** [comment]
> one comment, looks good otherwise. did you try creating these to make sure syntax was correct?

Yes, I tested the yaml files and got no syntax errors.

**Comment by pjbgf** [comment]
> /retest
> Is this something we want to land for v1.19?

This would be a nice to have.

File: cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml (hunk 1 above)
Comment (pjbgf) [reply cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml]: @liggitt @spiffxp PTAL

**Comment by liggitt** [comment]
looks like https://prow.k8s.io/job-history/gs/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-e2e-gce-network-proxy-grpc is solidly failing

/skip

**Comment by liggitt** [comment]
opened https://github.com/kubernetes/test-infra/pull/19033 to make the failing job optional

**Pull request merged**